roxie/ccd/ccdmain.cpp

```diff
@@ -1130,13 +1129,16 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                 if (port)
                 {
                     const char *protocol = roxieFarm.queryProp("@protocol");
-                    const char *passPhrase = nullptr;
-                    const char *certFile = nullptr;
-                    const char *keyFile = nullptr;
+                    const char *certFilePtr = nullptr;
```

Comment:
You don't need all of these xPtr variables.  Just use certFileName.str() when you pass it as a parameter to the createListener factory,